ansi.py

```diff
@@ -126,8 +157,10 @@ def plugin_loaded():
         generate_color_scheme(cs_file)
     settings = sublime.load_settings("ansi.sublime-settings")
     settings.add_on_change("ANSI_COLORS_CHANGE", lambda: generate_color_scheme(cs_file))
+    settings.add_on_change("ANSI_SETTINGS_CHANGE", lambda: AnsiColorBuildCommand.update_build_settings())
```

Comment:
Is this callback invoked when the setting are first loaded? If not, this will result in the ansi build setting being out of sync when sublime is restarted.